common/script/content/appearance/keys.js

```diff
@@ -0,0 +1,8 @@
+import {forOwn} from 'lodash';
+
+export default function(obj) {
```

Comment:
@vIiRuS I think the forOwn here is not cloning incoming objects so it's probably attaching `key` everywhere, you should first clone (`_.cloneDeep`) the object and then attach the `key` property